javalib/src/main/scala/java/io/Throwables.scala

```diff
@@ -17,3 +17,7 @@ class UTFDataFormatException(s: String) extends IOException(s) {
 class UnsupportedEncodingException(s: String) extends IOException(s) {
   def this() = this(null)
 }
+
+abstract class ObjectStreamException(s: String) extends IOException(s) {
+  def this() = this(null)
```

Comment:
The two constructors should be `protected`, according to [the JavaDoc](http://docs.oracle.com/javase/7/docs/api/java/io/ObjectStreamException.html)